autoload/lsp/omni.vim

```diff
@@ -321,6 +330,7 @@ function! lsp#omni#get_vim_completion_items(options) abort
             \ }
         if has_key(l:completion_item, 'textEdit') && type(l:completion_item['textEdit']) == s:t_dict && has_key(l:completion_item['textEdit'], 'newText')
```

Comment:
```suggestion
        if has_key(l:completion_item, 'textEdit') && type(l:completion_item['textEdit']) == s:t_dict && has_key(l:completion_item['textEdit'], 'newText') && has_key(l:completion_item['textEdit'], 'range')
```

I've got key not present error when I input like following(deno lsp).
```
const foo = 'foo';
foo.
```
After I enter `ifoo.<C-x><C-o>`, following message showed to log.
```
Fri Nov 12 00:45:48 2021:["s:on_stdout client request on_notification() error","Vim(let):E716: Key not present in Dictionary: \"range\"","function <SNR>139_out_cb[1]..<SNR>137_on_stdout[79]..<SNR>100_request_on_notification[3]..<SNR>101_createNext[1]..<SNR>101_subscribeSourceCallback[2]..<lambda>80[1]..<SNR>140_handle_omnicompletion[11]..<SNR>140_get_completion_result[9]..lsp#omni#get_vim_completion_items, line 38"]
```

So, should we check range key first?

Reply:
[InsertReplaceEdit]( https://microsoft.github.io/language-server-protocol/specifications/specification-3-17/#insertReplaceEdit) was returned.

Here is a log.